libvast/src/format/json.cpp

```diff
@@ -1040,7 +1040,8 @@ add(const ::simdjson::dom::object& object, table_slice_builder& builder) {
           },
           [&](const record_type& rt) {
             if (element.is_object()) {
-              self(self, element.get_object().value(), rt, "");
+              [[maybe_unused]] auto err
+                = self(self, element.get_object().value(), rt, "");
```

Comment:
We shouldn't be able to run into this error, as that would be a precondition violation. Please check `!err` with an assertion.